netcore/NOTICE

```diff
@@ -0,0 +1,6 @@
+Apache Geode
```

Comment:
Same as license, no need for notice here.

Reply:
removed